### ISSUE TITLE
Dark Mode Low Contrast Elements

### ISSUE THREAD

**Issue by drossano** [TheOdinProject/theodinproject#2538]
### Description
Certain elements in dark mode don't have enough contrast with their backgrounds:

- The `tracking-wide underline` class  on the [home page](https://www.theodinproject.com/) 
 ![brave_2022-01-20_20-26-28](https://user-images.githubusercontent.com/72749793/150452051-c48e0081-0576-4519-b723-9803a9ec47a2.png)
- The `course-card-header__sub-title` class on the [Ruby ](https://www.theodinproject.com/paths/full-stack-ruby-on-rails?)and [JS](https://www.theodinproject.com/paths/full-stack-javascript) path pages
![brave_2022-01-20_20-27-17](https://user-images.githubusercontent.com/72749793/150452104-3015d2b7-7dff-4c0d-9cf7-d4e27575731b.png)
- The `card-main profile-card` on the user dashboard.
![brave_2022-01-20_20-58-13](https://user-images.githubusercontent.com/72749793/150452141-27504b96-8a95-4803-a3ed-ab17bc517c3a.png)
![brave_2022-01-20_20-59-16](https://user-images.githubusercontent.com/72749793/150452149-f37de845-c55e-48b2-ab21-f871925dd9be.png)

- The bottom borders dividing the `skill`s on the user dashboard.
![brave_2022-01-20_20-58-53](https://user-images.githubusercontent.com/72749793/150452176-1aa32493-63ce-43a4-8b49-8d7af5ba0e5e.png)
![brave_2022-01-20_20-59-03](https://user-images.githubusercontent.com/72749793/150452186-6b866efd-0f8e-4ee2-9752-b606b531fc89.png)

#### Steps to Reproduce
Viewing the content in dark mode.
#### Expected Behavior
The content should have more contrast with its background

**Comment by DevlinRocha** [comment]
This is also present on the [FAQ page](https://www.theodinproject.com/faq), but only under the Code of Conduct question:

![cocOdin](https://user-images.githubusercontent.com/72491344/150702421-c252fc34-174d-47d2-89f7-7c89e719ae39.png)

**Comment by hoad211** [comment]
Can I take this one?

**Comment by ChargrilledChook** [comment]
Thanks to everyone who contributed with reports. All of these have now been fixed by updates in the Tailwind conversion
